packages/cli/state.json

```diff
@@ -1,9 +0,0 @@
-export default [fn((state) => {
```

Comment:
This is a random and unrelated fix - this file shouldn't be here and needs removing. Doing it here is the quickest way to main.